engine/src/main/java/io/camunda/zeebe/engine/processing/resource/ResourceDeletionProcessor.java

```diff
@@ -63,7 +63,12 @@ public void processCommand(final TypedRecord<ResourceDeletionRecord> command) {
   }
 
   @Override
-  public void processDistributedCommand(final TypedRecord<ResourceDeletionRecord> command) {}
+  public void processDistributedCommand(final TypedRecord<ResourceDeletionRecord> command) {
+    final var value = command.getValue();
+    deleteResources(command);
```

Comment:
🔧 In addition to the previous point, it's not clear what should happen here when `deleteResources` fails (it's highly unlikely and it really shouldn't, but it's possible). 

💭 Note that this is not clear anymore, now that the flow control is pushed down into that method and it did not have a name indicating that it could fail.

Reply:
Would you consider renaming to `tryDeleteResources` to be enough? I could of course make it return an `Either` but this comes with a tradeoff of duplicating the error handling, which I personally am not a fan of.

Reply:
❓ What should happen when the `tryDeleteResources` fails for a distributed command?

Reply:
We should reject the command. If we don't know the resource there is no need to delete it.
In theory this should only happen when the command is distributed twice. I suppose that's where the idempotency comes in. We must be able to handle this command. We do it here by rejecting it.